druid/examples/calc.rs

```diff
@@ -165,44 +178,79 @@ fn digit_button(digit: u8) -> impl Widget<CalcState> {
         .on_click(move |_ctx, data: &mut CalcState, _env| data.digit(digit))
 }
 
-fn flex_row<T: Data>(
+fn flex_row3<T: Data>(
+    w1: impl Widget<T> + 'static,
+    w2: impl Widget<T> + 'static,
+    w3: impl Widget<T> + 'static,
+) -> impl Widget<T> {
+    Flex::row()
+        .with_flex_child(Padding::new(1., w1), 2.0)
```

Comment:
Adding 1 pixel padding is different than what I had. What I'm going for is exactly 1 pixel between each button, but nothing around the outside edge. Here's the mac calculator for reference:
<img width="344" alt="Screen Shot 2020-03-23 at 10 43 33 AM" src="https://user-images.githubusercontent.com/543668/77328613-27466900-6cf3-11ea-936e-7c7ed4c87d70.png">

Reply:
yeah, i was trying to do that also, but after expanding the AC button to two columns, there is a 1-pixel mismatch with the button below AC.